rr_ml/nodes/semantic_segmentation/model_utils.py

```diff
@@ -0,0 +1,117 @@
+import keras
+import cv2
+import numpy as np
+
+
+class UNetModelUtils(object):
+    def __init__(self):
+        self.in_height = 256
+        self.in_width = 256
+
+        self.orig_width = None
+        self.orig_height = None
+
+        self.n_channels = 5
+        self.n_classes = 2
+
+    # significant credit to https://github.com/divamgupta/image-segmentation-keras
+    def make_unet_model(self):
```

Comment:
surprised I don't see any batchnorm in this architecture, but if this works then I guess you're fine.